backend/server/server.js

```diff
@@ -190,8 +193,9 @@ const server = http.createServer((request, response) => {
 });
 
 // We tell React Loadable to load all required assets and start listening.
-Loadable.preloadAll().then(() => {
+Loadable.preloadAll().then(async() => {
+    currentSite = await DotCMSApi.sites.getCurrentSite();
```

Comment:
I would like the input of @freddyucv and @hmoreras here, @alfredo-dotcms and I agreed to do this, but still feels weird to me.